### ISSUE TITLE
Remove jQuery

### ISSUE THREAD

**Issue by scottohara** [scottohara/tvmanager#79]
Over the years, the web platform has added new features such as
* `document.querySelector{All}()` for selecting DOM nodes
* `fetch()` for making network requests
* `Element.classList` for getting/setting CSS classes
etc.

We are now at the point where a lot of the reasons we used jQuery initially are no longer relevant, and we should consider removing this dependency in favour of using the native platform equivalents.

The benefit is one less dependency, which should save around ~30kb (gzipped) - ~85kb (uncompressed); as well as one less global (and the need to use `webpack.ProvidePlugin({jQuery: "jquery"})` in our bundles).

Before we can remove jQuery, however, we need to find a replacement for jQuery UI Sortable (https://github.com/scottohara/tvmanager/issues/78)

**Comment by scottohara** [comment]
As part of the migration to IndexedDb, we replaced `$.ajax()` calls with `fetch()` calls.
